packages/storybook/stories/va-combo-box-uswds.stories.jsx

```diff
@@ -46,6 +46,27 @@ const defaultArgs = {
   ],
 };
 
+const optGroupArgs = {
```

Comment:
Please move this down to just before the `export const OptGroups` below so that it's more obvious that these args go with that template.

Reply:
👍🏻